pkg/driver/postgres/postgres.go

```diff
@@ -361,6 +363,11 @@ func (drv *Driver) Ping() error {
 	return err
 }
 
+func (drv *Driver) IncreaseStatementTimeout(db *sql.DB, timeout time.Duration) error {
+	_, err := db.Exec(fmt.Sprintf("SET statement_timeout = %s", strconv.Itoa(int(timeout.Milliseconds()))))
```

Comment:
If this can be changed by a SQL statement, then does it really make sense to implement this specially in dbmate?  Why not include the statement in your migration file around the statement that requires the different statement timeout amount?

Reply:
because we usually run these in CI/CD and passing a flag to dbmate to take care of timeout is a little bit easier

Reply:
Are you saying that when you run these outside of CI/CD, changing the `statement_timeout` is unnecessary?

Is there any harm in baking the `SET statement_timeout` statements into your migrations that need it?  I would assume that only specific migrations that are long-running will need it and not every single migration that you're running, so having an app-wide setting seems like the wrong approach, anyway.

Reply:
The migration is originally written by a developer on their laptop. The statement timeout is an operational characteristic that might be different in a different environment. If the migration is under version control than these operational characteristics would have to be committed ahead of time for what the CI and production needs.